__tests__/app.test.js

```diff
@@ -22,3 +22,27 @@ describe("Get Categories", () => {
 			});
 	});
 });
+
+describe("Get Reviews", () => {
+	test("Return reviews and a status code 200", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then((result) => {
+				const expected = {
+					title: "Agricola",
+					designer: "Uwe Rosenberg",
+					owner: "mallionaire",
+					review_id: 1,
+					review_img_url:
+						"https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700",
+					review_body: "Farmyard fun!",
+					category: "euro game",
+					created_at: "2021-01-18T10:00:20.514Z",
+					votes: 1,
+				};
+				expect(result.body.reviews[0]).toEqual(expected);
+				expect(result.body.reviews.length).toBe(13);
+			});
+	});
```

Comment:
Something else we are missing here is that reviews should contain their comment count too. We could achieve this with a join in our query